pages/1.11/deploying-services/marathon-on-marathon/advanced/index.md

```diff
@@ -10,70 +10,95 @@ enterprise: true
+| `${MARATHON_TAG}` | The Docker image tag of the Marathon version that you want to deploy. For example `v1.5.11_1.10.2` (version 0.11.0 or newer).  |
 
-    ```
-    docker images
-    ```
+**Note:** If you are working on a MacOS or Linux machine, you can pre-define all the above variables in your terminal session and just copy and paste the snippets in your terminal:
 
-    You should see output similar to this:
+```
+set -a
+MY_USER_ID="..."
+AGENT_ID="..."
+MESOS_ROLE="..."
+SERVICE_ACCOUNT_ID="..."
+BEGIN_PORT="..."
+END_PORT="..."
+MARATHON_INSTANCE_NAME="..."
+SERVICE_ACCOUNT_SECRET="..."
+DOCKER_REGISTRY_SECRET="..."
+PRIVATE_KEY="..."
+PUBLIC_KEY="..."
+MARATHON_IMAGE="..."
+MARATHON_TAG="..."
+```
 
-    ```bash
-    REPOSITORY                    TAG                 IMAGE ID            CREATED             SIZE
-    mesosphere/marathon-dcos-ee   1.4.0-RC4_1.9.4     d1ffa68a50c0        3 months ago        926.4 MB
-    ```
 
-1. Rename the file to match the repository that you are using in your private Docker registry, where:
-   - `<mesosphere-tag>` is the tag of the image from Mesosphere. Typically, this will match the version number in the filename.
-   - `<your-repo>` is the name of the private repository that you want to store the image in.
-   - `<your-tag>` is the tag for the image. It is recommended that you use the same tag as the Mesosphere image.
+# Step 1: Preparation
 
-   ```bash
-   docker tag mesosphere/marathon-dcos-ee:<mesosphere-tag> <your-repo>/marathon-dcos-ee:<your-tag>
-   ```
+For the following steps, we are assuming that you have already:
 
-1. Push the Marathon image up to your private Docker registry.
+1. Pushed the enterprise image of Marathon to your private registry [(Instructions)](/1.11/deploying-services/private-docker-registry#tarball-instructions), under the name `${MARATHON_IMAGE}:${MARATHON_TAG}`.
+1. Stored your private Docker credentials in the secrets store [(Instructions)](/1.11/deploying-services/private-docker-registry/#referencing-private-docker-registry-credentials-in-the-secrets-store-enterprise), under the name `${DOCKER_REGISTRY_SECRET}`.
 
-   ```bash
-   docker push <your-repo>/marathon-dcos-ee:<your-tag>
-   ```
+    **Warning:** The name of the secret should either be in the root path (ex. `/some-secret-name`) or prefixed with the name of your app (ex. `/${MARATHON_INSTANCE_NAME}/some-secret-name`). Failing to do so, will make root Marathon unable to read the secret value and will fail to launch your custom Marathon-on-Marathon instance.
+
+# Step 2: Reserve Resources (Optional) {.tabs}
+In this step, we are going to define a [Mesos Role](https://mesos.apache.org/documentation/latest/roles/) for our new Marathon instance, and reserve some dedicated resources for it. This step is optional, but it's recommended if you want your new marathon instance to have a minimum resource guarantee for it's tasks.
```

Comment:
s/it's/it is/g

Reply:
Done. And also, `it's tasks` should be `its tasks`. And cap `Marathon`.

Reply:
✅